next/pages/form/[eform].tsx

```diff
@@ -68,15 +68,23 @@ const FormTestPage = ({
   const menuItems = mainMenu ? parseMainMenu(mainMenu) : []
   const router = useRouter()
 
+  let escapedSlug = ''
   const formSlug = forceString(router.query.eform)
-  const pageSlug = `form/${formSlug}`
+  // eslint-disable-next-line unicorn/prefer-regexp-test
+  if (formSlug.match(/^[\da-z-]+$/)) {
```

Comment:
why not comply to eslint ? if rewriting to using the suggested way brings other errors please add comment detailing